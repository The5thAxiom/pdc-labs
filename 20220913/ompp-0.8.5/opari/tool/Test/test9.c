int main() {
  int i;

#pragma \
 /*--*/ pomp \
 /*--*/ inst \
 /*--*/ init

#pragma pomp inst off

#pragma pomp inst begin(foo)

#pragma omp parallel
      i = 9;

#pragma pomp inst end(foo)

#pragma pomp inst on

#pragma pomp inst finalize

}
