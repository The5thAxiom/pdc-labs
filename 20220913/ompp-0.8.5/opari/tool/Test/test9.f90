      program test9

C$POMP  INST
C$POMP& INIT

      !$POMP INST OFF

      !$POMP INST &
      !$POMP BEGIN(foo)

      !$OMP PARALLEL
      i = 9
      !$OMP end PARALLEL

      !$POMP INST END(foo)

      !$POMP INST ON

   !$POMP INSTFINALIZE

      end
