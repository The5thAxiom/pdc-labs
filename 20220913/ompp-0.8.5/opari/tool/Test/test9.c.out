#ifdef _POMP
#  undef _POMP
#endif
#define _POMP 200110

#include "test9.c.opari.inc"
#line 1 "test9.c"
int main() {
  int i;

POMP_Init();
#line 8 "test9.c"

POMP_Off();
#line 10 "test9.c"

POMP_Begin(&omp_rd_24);
#line 12 "test9.c"

POMP_Parallel_fork(&omp_rd_25);
#line 13 "test9.c"
#pragma omp parallel POMP_DLIST_00025
{ POMP_Parallel_begin(&omp_rd_25);
#line 14 "test9.c"
      i = 9;
POMP_Barrier_enter(&omp_rd_25);
#pragma omp barrier
POMP_Barrier_exit(&omp_rd_25);
POMP_Parallel_end(&omp_rd_25); }
POMP_Parallel_join(&omp_rd_25);
#line 15 "test9.c"

POMP_End(&omp_rd_24);
#line 17 "test9.c"

POMP_On();
#line 19 "test9.c"

POMP_Finalize();
#line 21 "test9.c"

}
